create.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz App / create</title>
+    <link rel="stylesheet" href="./styles/index.css" />
+  </head>
+  <body>
+    <div class="app-grid">
+      <header><h1>Create a Question</h1></header>
+      <form action="" class="app-grid__scrollarea h-100vh" autocomplete="off">
```

Comment:
I am not sure if it is possible to include a section inside a form but maybe I am wrong ;)

Reply:
i am not sure too, but i will  try it